samples/algorithms/integer-factorization/Utils.qs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.IntegerFactorization {
+    open Microsoft.Quantum.Arithmetic;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Intrinsic;
+
+    
+    operation ApplyXorInPlaceL(value : BigInt, target : LittleEndian) : Unit is Adj+Ctl {
+        let bits = BigIntAsBoolArray(value);
+        let bitsPadded = Length(bits) > Length(target!) ? bits[...Length(target!) - 1] | bits;
+        ApplyPauliFromBitString(PauliX, true, bitsPadded, (target!)[...Length(bitsPadded) - 1]);
+    }
+
+    internal function TrailingZeroes(number : BigInt) : Int {
+        mutable zeroes = 0;
+        mutable copy = number;
+        while (copy % 2L == 0L) {
+            set zeroes += 1;
+            set copy /= 2L;
+        }
+        return zeroes;
+    }
```

Comment:
```suggestion
    internal function NTrailingZeroes(number : BigInt) : Int {
        mutable nZeroes = 0;
        mutable copy = number;
        while (copy % 2L == 0L) {
            set nZeroes += 1;
            set copy /= 2L;
        }
        return nZeroes;
    }
```

Reply:
I applied this in my commit for all comments.